src/applications/rated-disabilities/components/RatingList/index.jsx

```diff
@@ -0,0 +1,3 @@
+import RatingList from './RatingList';
```

Comment:
I could have included the `RatingList` component code in here, but I wanted to have the `RatingList` code be in a file named `RatingList.jsx` so it's easier to find